swagger-parser/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for swagger-parser 4.x
+// Project: https://www.npmjs.com/package/swagger-parser
+// Definitions by: Tobias Wolff <https://github.com/Tobias4872/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace SwaggerParser {
+  function parse(path: string): any;
+}
+
+export = SwaggerParser;
```

Comment:
That is right. I have done a bit more code reviewing and found not only options, but also a callback parameter. I will provide the changes in a minute.